### PR TITLE
fix: Broken styles in the editor

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

Fixes https://github.com/chatwoot/chatwoot/issues/5175

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

**Screenshot**

| **Before**                                                                                                                                     | **After**                                                                                                                                      |
| ------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------ |
| Screenshot 2022-08-03 at 8 52 31 AM | Screenshot 2022-08-03 at 8 57 00 AM |

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/components/widgets/conversation/ReplyEmailHead.vue [hunk 1]
```diff
@@ -117,6 +117,8 @@ export default {
   color: var(--r-500);
 }
 .input-group {
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/ReplyEmailHead.vue:119]: @iamsivin I don't think this is a correct fix. This issue came from https://github.com/chatwoot/chatwoot/pull/5103/files#diff-fe5d6ca412a0ed9343c2a796a0de32177375af5653e9ac0e2252d6cc8c004d00R67

Let us revert this change or update it properly. 

cc: @muhsin-k

Comment (iamsivin) [reply app/javascript/dashboard/components/widgets/conversation/ReplyEmailHead.vue:119]: Yeah will check it and update

Comment (iamsivin) [reply app/javascript/dashboard/components/widgets/conversation/ReplyEmailHead.vue:119]: @pranavrajs Updated https://github.com/chatwoot/chatwoot/pull/5183/commits/8503e1edd4f3a61ed636743e88579e15c13a0996 . Can you take a look?
